packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -50,7 +51,12 @@ function addVNodeToChildLWC(vnode: VCustomElement) {
 }
 
 // [h]tml node
-function h(sel: string, data: VElementData, children: VNodes = EmptyArray): VElement {
+function h(
+    renderer: RendererAPI,
```

Comment:
I placed as first argument hoping that we always pass it, but I realized mid-way there that it is not really needed for many functions in this file.